# Orthographic Image

sensor_msgs/Image image

float32 pixel_size
float32 min_depth
float32 max_depth

string camera
